pytorch_lightning/trainer/evaluation_loop.py

```diff
@@ -126,6 +142,7 @@ def setup(self, max_batches: List[Union[int, float]], dataloaders: List[DataLoad
         self.num_dataloaders = self._get_num_dataloaders(dataloaders)
 
     def on_evaluation_epoch_start(self, *args: Any, **kwargs: Any) -> None:
+        self.trainer.logger_connector.on_epoch_start()
```

Comment:
```suggestion
        # update ResultCollection.
        self.trainer.logger_connector.on_epoch_start()
```

Reply:
This doesn't update the `ResultCollection`, just sets a flag

I don't think that comment is useful